func/func_client_impl.cc

```diff
@@ -195,4 +198,33 @@ namespace func {
     return status.ok();
   }
 
+  // Helper method used by Event to handling streaming warbles.
+  bool FuncClientImpl::StreamHelper(int event_type,
+                    ClientEventParams& client_event_params, ClientEventReply& client_event_reply) {
+    EventRequest event_request;
+    EventReply event_reply;
+    ClientContext event_context;
+    google::protobuf::Any any_request;
+    google::protobuf::Any any_reply;
+    event_request.set_event_type(event_type);
+
+    // Construct stream request and reply. Make function call to func server.
+    StreamRequest stream_request;
+    StreamReply stream_reply;
+    stream_request.set_hashtag(client_event_params.hashtag);
+
+    any_request.PackFrom(stream_request);
+    event_request.mutable_payload()->google::protobuf::Any::MergeFrom(any_request);
+    event_reply.mutable_payload()->google::protobuf::Any::MergeFrom(any_reply);
+    Status status = stub_->event(&event_context, event_request, &event_reply);
+    event_reply.payload().UnpackTo(&stream_reply);
+    std::cout << "size func client " << stream_reply.warbles().size() << std::endl;
```

Comment:
You may want to delete this printing message. Printing warbles will be enough I think.

Reply:
Oh yes, this was for debugging!